sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/query.sql

```diff
@@ -329,13 +329,305 @@ desktop AS (
+    client_info.client_id
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_focus.baseline`
+  CROSS JOIN
+    UNNEST(ping_info.experiments) AS e
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+),
+exp_org_mozilla_focus_nightly AS (
+  SELECT DISTINCT
+    DATE(submission_timestamp) AS submission_date,
+    e.key AS experiment_id,
+    e.value.branch AS branch,
+    client_info.client_id
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_focus_nightly.baseline`
+  CROSS JOIN
+    UNNEST(ping_info.experiments) AS e
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+),
+exp_org_mozilla_focus_beta AS (
+  SELECT DISTINCT
+    DATE(submission_timestamp) AS submission_date,
+    e.key AS experiment_id,
+    e.value.branch AS branch,
+    client_info.client_id
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_focus_beta.baseline`
+  CROSS JOIN
+    UNNEST(ping_info.experiments) AS e
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+),
+exp_org_mozilla_ios_klar AS (
+  SELECT DISTINCT
+    DATE(submission_timestamp) AS submission_date,
+    e.key AS experiment_id,
+    e.value.branch AS branch,
+    client_info.client_id
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_ios_klar.baseline`
+  CROSS JOIN
+    UNNEST(ping_info.experiments) AS e
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+),
+exp_org_mozilla_ios_focus AS (
+  SELECT DISTINCT
+    DATE(submission_timestamp) AS submission_date,
+    e.key AS experiment_id,
+    e.value.branch AS branch,
+    client_info.client_id
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_ios_focus.baseline`
+  CROSS JOIN
+    UNNEST(ping_info.experiments) AS e
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+),
+experiments_information AS (
+  SELECT
+    *
+  FROM
+    (
+      SELECT
+        *
+      FROM
+        exp_org_mozilla_firefox_beta
+      UNION ALL
+      SELECT
+        *
+      FROM
+        exp_org_mozilla_fenix
+      UNION ALL
+      SELECT
+        *
+      FROM
+        exp_org_mozilla_firefox
+      UNION ALL
+      SELECT
+        *
+      FROM
+        exp_org_mozilla_ios_firefox
+      UNION ALL
+      SELECT
+        *
+      FROM
+        exp_org_mozilla_ios_firefoxbeta
+      UNION ALL
+      SELECT
+        *
+      FROM
+        exp_org_mozilla_ios_fennec
+      UNION ALL
+      SELECT
+        *
+      FROM
+        exp_telemetry
+      UNION ALL
+      SELECT
+        *
+      FROM
+        exp_org_mozilla_klar
+      UNION ALL
+      SELECT
+        *
+      FROM
+        exp_org_mozilla_focus
+      UNION ALL
+      SELECT
+        *
+      FROM
+        exp_org_mozilla_focus_nightly
+      UNION ALL
+      SELECT
+        *
+      FROM
+        exp_org_mozilla_focus_beta
+      UNION ALL
+      SELECT
+        *
+      FROM
+        exp_org_mozilla_ios_klar
+      UNION ALL
+      SELECT
+        *
+      FROM
+        exp_org_mozilla_ios_focus
+    )
+  WHERE
+    submission_date = @submission_date
```

Comment:
This filter is required? The subqueries already filter the data for that submission_date.